src/plugins/elasticsearch/lib/__tests__/check_es_version.js

```diff
@@ -73,14 +73,16 @@ describe('plugins/elasticsearch', function () {
       }, _.noop);
     });
 
-    it('passes if that single node is a client node', function () {
+    it('fails if that single node is a client node', function () {
       setNodes(
         '1.4.4',
         { version: '1.4.2', attributes: { client: 'true' } },
         '1.4.5'
       );
-
-      return checkEsVersion(server);
+      return checkEsVersion(server)
+      .then(function() {
+        throw new Error('expected validation to fail')
+      }, _.noop);
```

Comment:
Hm. Looks like the example copied of a fail-check could be improved. It seems like it works because of an unhandled error, and the test is made to fail if there's no error that slips by.

A more typical way to test that a function throws an error is something like:

```javascript
function checkWithBadClientNode() {
  return checkEsVersion(server);
}
expect(checkWithBadClientNode).to.throwException(function (e) { // get the exception object
  expect(e).to.be.a(SetupError);
});
```

That kind of test will be possible if you import `expect` and `SetupError` into the suite. I prefer this way because it is more explicit what kind of error gets thrown for the execution path you want to test.